src/hooks/useNotificationSound.ts

```diff
@@ -0,0 +1,11 @@
+import { useEffect } from "react";
+import notificationSound from "assets/sounds/notification.m4a";
+
+export const useNotificationSound = (numberOfUnreadMessages: number) => {
+
+  useEffect(() => {
+    if (numberOfUnreadMessages <= 0) return
+
+    new Audio(notificationSound).play();
+  }, [numberOfUnreadMessages]);
```

Comment:
@mike-lvov / @sparkletown/engineering What do you think about this part? It will re-create the `Audio` object every time this effect is called. Do you know if that will cause it to need to re-fetch the sound file/etc? Or is it smart enough to know how to handle that?

Should we be creating this once and keeping that 'saved'/memo'd, and then just calling `.play()` on it each time it's needed?

Reply:
On further reflection, maybe it's better to leave it as it is. As it currently is, while it will create the object each time (which will put some pressure on the garbage collector), it won't hold the audio asset in memory indefinitely.

If we changed it, we would hold the audio asset in memory, even if we never actually get a private message that would trigger this sound effect to play.